### DISCUSSION TITLE
Welcome to unzipLIB Discussions!

### DISCUSSION THREAD

**Discussion by bitbank2** [bitbank2/unzipLIB#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RDZDX** [comment]
Hello,
how to save unziped data as file ?

**Comment by bitbank2** [comment]
That's up to you. This library can get you the data from a zip file and you manage it from there.
